src/main/scala/nl.knaw.dans.easy.multideposit/package.scala

```diff
@@ -114,8 +114,8 @@ package object multideposit {
      */
     @throws[IOException]("in case of an I/O error")
     def writeXml(elem: Elem, encoding: Charset = encoding): Unit = {
-      Files.createDirectories(file.getParent)
-      XML.save(file.toString, elem, encoding.toString, xmlDecl = true)
+      Files.createDirectories(path.getParent)
+      XML.save(path.toString, XML.loadString(new PrettyPrinter(160, 2).format(Utility.trim(elem))), encoding.toString, xmlDecl = true)
```

Comment:
It is quite a detour to convert the string back to xml and implicitly to a string again just to add the prologue. Can't it be added with some constant, or would that be too error prone?